src/Console/User.php

```diff
@@ -63,6 +64,13 @@ protected function getHelp()
 	bin/console user deny [<nickname>] [-h|--help|-?] [-v]
 	bin/console user block [<nickname>] [-h|--help|-?] [-v]
 	bin/console user unblock [<nickname>] [-h|--help|-?] [-v]
+	bin/console user list pending [start=0 [count=50]] [-h|--help|-?] [-v]
```

Comment:
Shouldn't start and count be named parameters like `--start` or `-s` and `--count` or `-n` instead of positional arguments?

Reply:
mhmm .. good idea